### ISSUE TITLE
When running a simple gzip interface suddenly a tk window appears (Ubuntu 14.10)

### ISSUE THREAD

**Issue by Garyfallidis** [nipy/nipype#1281]
Hi all,

I made a simple interface file like this one copied from the tutorial found here
http://nipy.org/nipype/devel/cmd_interface_devel.html

Here how it looks in detail
```
from nipype.interfaces.base import (
    TraitedSpec,
    CommandLineInputSpec,
    CommandLine,
    File
)

import os

class GZipInputSpec(CommandLineInputSpec):
    input_file = File(desc="File", exists=True, mandatory=True, argstr="%s")

class GZipOutputSpec(TraitedSpec):
    output_file = File(desc = "Zip file", exists = True)

class GZipTask(CommandLine):
    input_spec = GZipInputSpec
    output_spec = GZipOutputSpec
    cmd = 'gzip'

    def _list_outputs(self):
            outputs = self.output_spec().get()
            outputs['output_file'] = os.path.abspath(self.inputs.input_file + ".gz")
            return outputs

if __name__ == '__main__':

    zipper = GZipTask(input_file='test.txt')
    print zipper.cmdline
    zipper.run()
```

All good so far. However, when I run this file above a tk window appeared!. But this should only run gzip. 

The tk window appeared because in the same folder with "test.txt" there was also a "test.py" file which would open a tk window. This means that there is probably a bug in nipype that can execute input files. As you can imagine, I believe this can be very dangerous.

I called this nipype interface from the command line using the command

```
python simple_interface.py
```

I hope this information was helpful. I installed nipype using pip install. So, this bug should affect also the current release version.

**Comment by chrisfilo** [comment]
I've tried to replicate this (see https://gist.github.com/chrisfilo/0cd489059434389d3c71), but failed. Could you try afresh with my code? Or share the code and precise steps to replicate?

**Comment by Garyfallidis** [comment]
Hi @chrisfilo I have good news for you. The execution was happening only on a specific terminal. Maybe that terminal didn't have the environment variables correct. When I switched to a new console I stopped seeing the issue. So, I would suggest to close the issue for now. And I will report back if I see it again in my machine. But I think it was some setup issue and not related to nipype specifically.
